src/components/SideEffects.js

```diff
@@ -0,0 +1,64 @@
+import React from "react"
+
+import { darkgrey } from "../colors"
+import { observer } from "mobx-react"
+import { DateTime } from "luxon"
+import Fold from "../primitives/Fold"
+
+import { Icon } from "@mdi/react"
+import { mdiFormatListChecks } from "@mdi/js"
+import Heading from "../primitives/Heading"
+import { Paragraph } from "reakit";
+
+import styled from "styled-components"
+
+const SideEffects = observer(({ store }) => (
+      <div>
+        <Fold>
+          <Question>
+          <Icon size="1.5rem" color={darkgrey} path={mdiFormatListChecks} /> {store.translate("progress.side_effect")}
+          </Question>
+
+          { store.registration.information.symptom_reports.length === 0
+            ? <Answer>
+                {/* TODO: Translate this piece */}
+                <Info>No Side Effects Reported</Info>
+              </Answer>
+
+            : store.registration.information.symptom_reports.map(({created_at, reported_symptoms}) => (
+              
+              // TODO: Sort by date and link reports
+              <Answer>
+                <Time key={created_at}>{DateTime
+                .fromISO(created_at)
+                .toLocaleString(DateTime.DATETIME_SHORT)}</Time>
+                {reported_symptoms.length !== 0 ? 
+                <Info key={reported_symptoms}>{reported_symptoms.join(", ")}</Info>
+                  // TODO: Translations
+                : <Info>N/A</Info>}
```

Comment:
```suggestion
                : <Info>{store.translate("progress.no_side_effects")}</Info>
                }
```

This translation is already supplied by the previous comment; we can use the same wording in both places.

Reply:
I think it is a little lengthly but lets keep it for now